.github/workflows/botonic-react-tests.yml

```diff
@@ -31,4 +35,12 @@ jobs:
     - name: Verify lint
       run: (cd ./packages/$PACKAGE && npm run lint_ci)
     - name: Verify index.d.ts
-      run: scripts/qa/lint-d-ts.sh packages/botonic-core
+      run: scripts/qa/lint-d-ts.sh ./packages/$PACKAGE
+    
+    - name: Upload coverage to codecov
```

Comment:
can you try this example https://stackoverflow.com/a/59798779/145289  to see if you can have the codecov config in a single file?

Reply:
From what I have read (https://github.com/actions/runner/issues/646), that would the case for reusing a pure bash/docker/JS action, not to call one action from another.
I didn't tried because of that link but I can give it a try.